.pre-commit-config.yaml

```
@@ -14,3 +14,7 @@ repos:
     rev: v2.7.1
     hooks:
       - id: prettier
+  - repo: https://github.com/psf/black
+    rev: 23.1.0
```

```suggestion
    rev: 23.3.0
```
I think it makes sense to use the same version like in the pyproject.toml